packages/flutter/lib/src/material/toggleable.dart

```diff
@@ -50,6 +52,8 @@ abstract class RenderToggleable extends RenderConstrainedBox {
        _inactiveColor = inactiveColor,
        _hoverColor = hoverColor ?? activeColor.withAlpha(kRadialReactionAlpha),
        _focusColor = focusColor ?? activeColor.withAlpha(kRadialReactionAlpha),
+       _reactionColor = reactionColor ?? activeColor.withAlpha(kRadialReactionAlpha),
+       _inactiveReactionColor = inactiveReactionColor ?? activeColor.withAlpha(kRadialReactionAlpha),
```

Comment:
Should this use `inactiveColor.withAlpha(kRadialReactionAlpha)` as the fallback instead?

Reply:
That would break internal Google tests as it's a change from what it was before (screenshot tests that were added by me).

But it would probably follow the Material specs better to use the `inactiveColor`. I think changing it to use `inactiveColor` can be done after this has been merged. Do you agree?

Reply:
gotcha, yeah that seems reasonable to me